docs/topics/data-science-overview.md

```diff
@@ -118,6 +118,8 @@ library for Java
 
 * [Charts](https://github.com/HanSolo/charts) - a scientific JavaFX charting library in development
 
+* [Apache OpenNLP](https://opennlp.apache.org/) - a machine learning based toolkit for the processing of natural language text
```

Comment:
```suggestion
* [Apache OpenNLP](https://opennlp.apache.org/) - a machine learning based toolkit for the processing of a natural language text
```

Reply:
I don't think it's necessary to change this. The word "text" can be uncountable, and this sentence is directly copied from [OpenNLP's homepage](https://opennlp.apache.org/).